src/pages/uxp-api/reference-js/Global Members/HTML DOM/Alerts/confirm.md

```diff
@@ -0,0 +1,26 @@
+
+<a name="confirm" id="confirm"></a>
```

Comment:
@kareemmyk  Possible to move alert.md, confirm and prompt within an 'Alerts' folder in torq-native?

Reply:
We did investigate this option.
This is a limitation of the source code for docs generation. We can have a parent and a child folder but no additional folder inside of it. In this case, it's Global Members(parent) -> HTML DOM(child).
So at this stage the current state is our best option